roles/confluent.variables/defaults/main.yml

```diff
@@ -35,11 +35,11 @@ jmxexporter_enabled: false
 jmxexporter_jar_path: /opt/prometheus/jmx_prometheus_javaagent.jar
 
 ### Boolean to have cp-ansible configure components with FIPS security settings
-fips_enabled: false
+fips_enabled: "{{_fips_enabled}}"
 
 fips_jar_path: /usr/share/java/kafka/bc-fips-*.jar
 fips_provider_class: org.bouncycastle.jcajce.provider.BouncyCastleFipsProvider
-pkcs12_enabled: "{{ true if fips_enabled|bool else false }}"
+pkcs12_enabled: "{{ _pkcs12_enabled }}"
```

Comment:
i think i meant to delete these in the fips rework... at the minimum i do not think you need to pay any attention to pkcs12_enabled

Reply:
It's removed in 6.2.x but not in 6.1.0-post where I'm basing the work from